python/src/scipp/plotting/model.py

```diff
@@ -229,6 +227,13 @@ def _make_profile(self, profile, dim, mask_info):
         values["masks"] = self._make_masks(profile, mask_info=mask_info)
         return values
 
+    def update_data_arrays(self):
+        for name, item in self.backup.items():
+            self.data_arrays[name] = sc.DataArray(
+                data=item["array"].data,
+                coords=item["coords"],
+                masks=item["array"].masks.to_dict())
```

Comment:
Does not appear to use `self.name`, so I do not understand the comment above.

Reply:
The overload of `update_data_arrays` in `model2d.py` does make use of `self.name`.